src/index.js

```diff
@@ -346,6 +346,32 @@ export default function({ types: t }) {
         fixReactNativeViewAttributes(path);
         fixTouchableMixin(t, path);
       },
+      JSXAttribute(path) {
```

Comment:
@JoshMiers-FS You mentioned only running this when we detect Fabric. Based on my research, it's not possible to detect Fabric in the babel plugin. We must ask the user to supply us with a config so that we know whether Fabric is `on` or `off`. What do you think about adding this config?

Reply:
Well, maybe it isn't an issue if we don't detect fabric. I'd hate to have another config that a customer could forget to set.